.devcontainer/devcontainer.json

```
@@ -0,0 +1,23 @@
+{
+	"name": "scikit-image devcontainer",
+	"build": {
+		"context": "..",
+		"dockerfile": "Dockerfile"
+	},
+	"features": {
+		"ghcr.io/devcontainers-contrib/features/pre-commit:2": {}
+	},
+	"postCreateCommand": "pre-commit install && bash ./.devcontainer/postBuild",
```

```suggestion
	"postCreateCommand": "bash ./.devcontainer/postBuild",
```
Just to be safe it may be good to use the `pre-commit` in from the dev environment.